### PR TITLE
Update mac-schedule-scan.md

### PR DESCRIPTION
**Pull request by msbemba**
Added more information  to clarify the values for StartCalendarInterval

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7379

File: microsoft-365/security/defender-endpoint/mac-schedule-scan.md [hunk 1]
```diff
@@ -127,10 +127,14 @@ The following code shows the schema you need to use to schedule a quick scan.
     launchctl start <your file name>
     ```
 
-3. Your scheduled scan will run at the date, time, and frequency you defined in your p-list. In the examples above, the scan runs at 2:00 AM every Friday. 
-
-    The `Weekday` value of `StartCalendarInterval` uses an integer to indicate the fifth day of the week, or Friday.
+3. Your scheduled scan will run at the date, time, and frequency you defined in your p-list. In the examples above, the scan runs at 2:50 AM every Friday. 
 
+    The `Weekday` value of `StartCalendarInterval` uses an integer to indicate the fifth day of the week, or Friday. The Range is between  0-7 and 7 represents sunday.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mac-schedule-scan.md]:
```suggestion
    - The `Weekday` value of `StartCalendarInterval` uses an integer to indicate the fifth day of the week, or Friday. The range is between 0 and 7 with 7 representing Sunday.
```

File: microsoft-365/security/defender-endpoint/mac-schedule-scan.md [hunk 2]
```diff
@@ -127,10 +127,14 @@ The following code shows the schema you need to use to schedule a quick scan.
...
+    The `Day` value of `StartCalendarInterval` uses an integer to indicate the third day of the month. The range is between 1-31.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mac-schedule-scan.md]:
```suggestion
    - The `Day` value of `StartCalendarInterval` uses an integer to indicate the third day of the month. The range is between 1 and 31.
```

File: microsoft-365/security/defender-endpoint/mac-schedule-scan.md [hunk 3]
```diff
@@ -127,10 +127,14 @@ The following code shows the schema you need to use to schedule a quick scan.
...
+    The `Hour` value of `StartCalendarInterval` uses an integer to indicate the second hour of the day. The range is between 0-24.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mac-schedule-scan.md]:
```suggestion
    - The `Hour` value of `StartCalendarInterval` uses an integer to indicate the second hour of the day. The range is between 0 and 24.
```

File: microsoft-365/security/defender-endpoint/mac-schedule-scan.md [hunk 4]
```diff
@@ -127,10 +127,14 @@ The following code shows the schema you need to use to schedule a quick scan.
...
+    The `Minute` value of `StartCalendarInterval` uses an integer to indicate Fivetly minutes of the Hour. The range is between 0-59.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mac-schedule-scan.md]:
```suggestion
    The `Minute` value of `StartCalendarInterval` uses an integer to indicate fifty minutes of the hour. The range is between 0 and 59.
```

File: microsoft-365/security/defender-endpoint/mac-schedule-scan.md [hunk 5]
```diff
@@ -127,10 +127,14 @@ The following code shows the schema you need to use to schedule a quick scan.
     launchctl start <your file name>
     ```
 
-3. Your scheduled scan will run at the date, time, and frequency you defined in your p-list. In the examples above, the scan runs at 2:00 AM every Friday. 
-
-    The `Weekday` value of `StartCalendarInterval` uses an integer to indicate the fifth day of the week, or Friday.
+3. Your scheduled scan will run at the date, time, and frequency you defined in your p-list. In the examples above, the scan runs at 2:50 AM every Friday.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mac-schedule-scan.md]:
```suggestion
3. Your scheduled scan will run at the date, time, and frequency you defined in your p-list. In the previous examples, the scan runs at 2:50 AM every Friday. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**